bacon/model/Transaction.swift

```diff
@@ -8,13 +8,29 @@
 
 import Foundation
 
-struct Transaction: Codable, Equatable {
-    let date: Date
-    let type: TransactionType
-    let frequency: TransactionFrequency
-    let category: TransactionCategory
-    let amount: Decimal
-    let description: String
+/// Represents a mutable transaction.
+class Transaction: Codable, Observable {
+
+    private var _date: Date
+    private var _type: TransactionType
+    private var _frequency: TransactionFrequency
+    private var _category: TransactionCategory
+    private var _amount: Decimal
+    private var _description: String
+
```

Comment:
why description? I thought we don't have description but just a picture which is NSData?

Reply:
I know we want an `image` property but are we replacing description? :O